.github/workflows/publish-documentation.yml

```diff
@@ -18,6 +18,21 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - id: getProjectPath
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const projectPath = function(gh_repo) {
+              let path = '';
+              gh_repo.topics.forEach((topic) => {
+                if (topic.startsWith('project-')) {
+                  path = topic.substring(8).toUpperCase() + '/';
```

Comment:
```suggestion
                  path = topic.substring(8).toLowerCase() + '/';
```

Reply:
oh
I put it in upper case to visually distinguish projects from repositories in the tree
But if you prefer them lower, I can simply remove the transformation as it is already lower case in the topic

Reply:
Maybe we can add a `project-` prefix or a `project` subdirectory instead?

Reply:
if we name folders with `project-` prefix, it will become much easier: the topic is already prefixed `project-pod`

Reply:
Let's go with it then 👌